src/apps/tabs/main.js

```diff
@@ -10,11 +12,49 @@ let React = require('react');
 
 let DefaultTab = require('./components/default-tab.react');
 
-// render the default tab
-let containerEl = document.createElement('div');
-document.body.insertBefore(containerEl, document.body.firstChild)
+/**
+ * Constants.
+ */
+
+const TAB_DISABLED_KEY = process.env.TAB_DISABLED_KEY;
+const GA_ID = process.env.GA_ID;
+
+/**
+ * Render the default tab.
+ */
+
+function render() {
+  let containerEl = document.createElement('div');
+  document.body.insertBefore(containerEl, document.body.firstChild)
+  React.render(<DefaultTab />, containerEl);
+}
+
+/**
+ * Load Google Analytics.
+ */
+
+function loadGa() {
+  var _gaq = window._gaq = _gaq || [];
+  _gaq.push(['_setAccount', GA_ID]);
+  _gaq.push(['_trackPageview']);
+
+  var ga = document.createElement('script');
+  ga.type = 'text/javascript';
+  ga.async = true;
+  ga.src = 'https://ssl.google-analytics.com/ga.js';
+  var s = document.getElementsByTagName('script')[0];
+  s.parentNode.insertBefore(ga, s);
+}
+
+let option = {};
+option[TAB_DISABLED_KEY] = false;
 
-React.render(
-  <DefaultTab />,
-  containerEl
-);
+chrome.storage.sync.get(option, function(items) {
```

Comment:
this `chrome.storage.sync` is used in several places. I think you can have a module just name `Cache` or similar.